packages/components/src/components/primitives/Button/Button.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { Button } from './Button';
+import { HStack, StackDivider, VStack } from '@chakra-ui/layout';
+import { SIZES } from '../../../theme/utils/constants';
+import { LIST_VARIANTS } from './constants';
+
+export default {
+  component: Button,
+  title: 'Components/Primitives/Button',
+} as Meta;
+
+const AllTemplate = () => (
+  <VStack divider={<StackDivider borderColor="gray.200" />} spacing={4}>
+    {LIST_VARIANTS.map((value) => (
+      <VStack key={value}>
+        <HStack>
+          {SIZES.map((size) => (
+            <Button variant={value} size={size} key={size}>
+              {value}
+            </Button>
+          ))}
+        </HStack>
+        <HStack key={value}>
+          {SIZES.map((size) => (
+            <Button variant={value} size={size} key={size} disabled>
+              {value}
+            </Button>
+          ))}
+        </HStack>
+      </VStack>
+    ))}
+  </VStack>
+);
+
+export const All = AllTemplate.bind({});
```

Comment:
in general we add a `Template` component that we can interact with in the storybook

Reply:
added the template component to interact with the button